cmd/cniplugins/passt-binding/cmd/BUILD.bazel

```diff
@@ -14,7 +14,7 @@ go_library(
 )
 
 go_binary(
-    name = "kubevirt-passt-binding",
+    name = "network-passt-binding",
```

Comment:
I think it will break existing users of the passt binding plugin (and our docs).
@EdDev WDYT?